sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamCalcRelTest.java

```diff
@@ -213,17 +212,14 @@ public void testSingleFieldAccess() throws IllegalAccessException {
 
     ParDo.MultiOutput<Row, Row> pardo =
         (ParDo.MultiOutput<Row, Row>) nodeGetter.producer.getTransform();
-    DoFnSignature sig = DoFnSignatures.getSignature(pardo.getFn().getClass());
+    PCollection<Row> input =
+        (PCollection<Row>) nodeGetter.producer.getInputs().values().iterator().next();
```

Comment:
It may be better to use `Iterables.getOnlyElement` instead. (likewise for other tests)

Reply:
Done.